### DISCUSSION TITLE
Review

### DISCUSSION THREAD

**Discussion by galalkoro98** [HYF-Class19/agile-development-loruki-group-1#2] [Announcements]
To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sammou00** [comment]
Hello, 
Well done.
Here are some notes :
1- user personas are plurals 
![image](https://user-images.githubusercontent.com/67711948/182943591-c2d30e53-cc87-4bd1-a9f2-2049a332d29b.png)

2- 1 week deadline is an external  constrain
![image](https://user-images.githubusercontent.com/67711948/182943869-477a423d-9813-40ba-a150-338f814ad46b.png)

3-  only web developers want to see the home page showcase or all user-personas ?
![image](https://user-images.githubusercontent.com/67711948/182944106-ce2630fe-7e15-4dfa-9880-8b73d0a45d37.png)

4- in design normally we don't show html tags only design 
![image](https://user-images.githubusercontent.com/67711948/182944347-72e5ce3e-54a9-4546-b6e7-d33581466d27.png)


5- good strategy-plan , Well done.

6- can't see anything in `index.html` and `index.css` ??

**Reply by galalkoro98** [reply]
@samirm00 yes indeed but now there is activity on html files & css as well, if you check our project board you can get full view on our groub project progress
